data.json

```
@@ -1324,5 +1324,11 @@
     "githubUsername": "Krishnapro",
     "imageUrl": "https://user-images.githubusercontent.com/59277887/132136661-d62a6664-0fa3-437d-b97d-ea41ca21c61f.png",
     "issueNumber": 702
-  }
+  },
+  {
+    "name": "Omkar Kulkarni",
+    "githubUsername": "Omkar0114",
+    "imageUrl": "https://user-images.githubusercontent.com/51391473/133314407-008ec664-bdbd-4428-a9c4-8c306f74a91c.png",
+    "issueNumber": 713
+  },
```

Remove the comma here, it is not needed.
```suggestion
  }
```